bin/stats19

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env perl
+use strictures;
+
+use lib 'perllib';
+use Dataset::UK::Stats19;
+
+Dataset::UK::Stats19->new_with_cmd->execute;
```

Comment:
This seems to be a lot of data/code for something that isn't core to FixMyStreet – would it be better placed in a separate repo?